### ISSUE TITLE
Investigate memory override when returning chromosome

### ISSUE THREAD

**Issue by mikeheddes** [mikeheddes/dcgp.js#21]
I found a chromosome returned from `Expression.getChromosome` that contained pointers. Investigate how that could happen and how to prevent it.

**Comment by mikeheddes** [comment]
Closed with #22

**Comment by mikeheddes** [comment]
A more thorough investigation showed that that the bug is still there.

### Steps to reproduce
```js
let dcgp = await dcgp.createInstance(fetch('/dcgp.wasm'))
let myKernelSet = new dcgp.KernelSet(['diff', 'mul', 'pdiv', 'sin', 'cos'])
let myExpression = new dcgp.Expression(2, 1, 1, 25, 5, 2, myKernelSet, 360);

let inputs = [[0, 1], [1, 1], [2, 1], [3, 1], [4, 1]]
let outputs = [[2], [4], [6], [8], [10]]

dcgp.algorithms.onePlusLambda(myExpression, 1, 0, inputs, outputs)
// restult.chromosome is
// [5253336, 5253336, 0, 0, 6412, 2, 2, 0, 3, 2, 4, 2, 1, 0, 2, 2, 5, 3, 2, 6, 3, 0, 8, 4, 3, 7, 9, 0, 8, 8, 0, 7, 11, 0, 9, 8, 4, 12, 13, 2, 12, 13, 2, 15, 15, 1, 12, 14, 3, 16, 15, 0, 17, 14, 1, 19, 17, 2, 20, 17, 0, 17, 18, 0, 20, 22, 4, 20, 23, 1, 21, 24, 0, 21, 25, 24]
```
